### ISSUE TITLE
contributor guide incorrect

### ISSUE THREAD

**Issue by ashconnell** [react-community/react-navigation#1325]
I followed the contributor guide and all i got was a packager failure `Unexpected token import`

But what did work was to:

1. Ensure you have node 7+ npm 4.5.0+
2. clone the repo
3. npm install
4. react-native start (start the packager manually)
5. cd example/NavigationPlayground
6. npm install
7. react-native run-ios

**Comment by IOriens** [comment]
Tried this method, got lots of `@providesModule naming collision` errors, but it finally run normally. Thank you man!

**Comment by oguennec** [comment]
This worked for example/ReduxExample although `react-native start` results in lots of warnings such as

> This warning is caused by a @providesModule declaration with the same name across two different files.
> jest-haste-map: @providesModule naming collision:
>   Duplicate module name: react-native-implementation
>   Paths: /Users/olivier/react-native/react-navigation/node_modules/react-native/Libraries/react-native/react-native-implementation.js collides with /Users/olivier/react-native/react-navigation/examples/ReduxExample/node_modules/react-native/Libraries/react-native/react-native-implementation.js

$ node --version
v7.10.0
$ npm --version
4.6.1

**Comment by oguennec** [comment]
The following issue seems to be the root cause of the problem:
[jest-haste-map: @providesModule naming collision](https://github.com/react-community/create-react-native-app/issues/193)

I could reproduce the error with the following versions:

- jest@20.0.1
- react-native@0.44.0

**Comment by oguennec** [comment]
Installing  **create-react-native-app** globally as below (which installs **react-native-scripts** as well) removes the warnings.

`npm install -g create-react-native-app`

**Comment by ajostergaard** [comment]
@oguennec **create-react-native-app** did not install **react-native-scripts** for me:

```
aj$ npm install -g create-react-native-app
npm WARN deprecated fs-promise@1.0.0: Use mz or fs-extra^3.0 with Promise Support
/usr/local/bin/create-react-native-app -> /usr/local/lib/node_modules/create-react-native-app/build/index.js
added 34 packages in 5.048s
aj$
```

**Comment by KevKo1990** [comment]
As an addition to @ashconnell initial's post for running android instead of iOS (did not check iOS)

- go to folder examples/NavigationPlayground
- do `react-native upgrade`
- then `react-native run-android`

without doing `react-native upgrade`, I received the error that the folder was not recognized as an Android project.

**Comment by KevKo1990** [comment]
I followed your instructions @ashconnell but for Android, it is not working... Folder /examples/NavigationPlayground is not recognized as Android project.

I use node version 7.10.0 and npm 4.6.1 on Ubuntu 16.04.

Then I read one could do the following:
- go into /examples/NavigationPlayground
- do `react-native upgrade`
- do `react-native run-android`

Now the app is installing on my emulator, however not working. Now, I receive the error the file index-android.js is missing.
I created in `react-navigation/` the file `index.android.js` with the following content:
```
import React from 'react';
import { AppRegistry } from 'react-native';
import { App } from './examples/NavigationPlayground/App';

AppRegistry.registerComponent('App', () => App);
```
and now... I receive more errors...
![screenshot_1497577778](https://user-images.githubusercontent.com/12497609/27208591-81e820e0-5281-11e7-8048-905ec62145a5.png)

Does someone has an idea how to get it runnable on Android?
Cheers!
